scan/lib/scan/test_command_generator.rb

```diff
@@ -95,6 +95,14 @@ def xcodebuild_log_path
         return File.join(containing, file_name)
       end
 
+      # Generate destination parameters
+      def destination
+        unless Scan.cache[:destination]
+          Scan.cache[:destination] = [*Scan.config[:destination]].map { |dst| "-destination '#{dst}'" }.join(' ')
```

Comment:
Consider `Array(Scan.config[:destination])` as it feels a little less tricky to me, and works if `Scan.config[:destination]` is an Array, and also works if it is a String.